collectors/logsmanagement.plugin/plugin_logsmanagement_web_log.h

```diff
@@ -0,0 +1,135 @@
+    *dim_req_method_link, *dim_req_method_lock, *dim_req_method_merge, *dim_req_method_mkactivity, *dim_req_method_mkcalendar, *dim_req_method_mkcol,
+    *dim_req_method_mkredirectref, *dim_req_method_mkworkspace, *dim_req_method_move, *dim_req_method_options, *dim_req_method_orderpatch, 
+    *dim_req_method_patch, *dim_req_method_post, *dim_req_method_pri, *dim_req_method_propfind, *dim_req_method_proppatch, *dim_req_method_put,
+    *dim_req_method_rebind, *dim_req_method_report, *dim_req_method_search, *dim_req_method_trace, *dim_req_method_unbind, *dim_req_method_uncheckout,
+    *dim_req_method_unlink, *dim_req_method_unlock, *dim_req_method_update, *dim_req_method_updateredirectref;
+
+    collected_number num_req_method_acl, num_req_method_baseline_control, num_req_method_bind, num_req_method_checkin, num_req_method_checkout,
+    num_req_method_connect, num_req_method_copy, num_req_method_delet, num_req_method_get, num_req_method_head, num_req_method_label, 
+    num_req_method_link, num_req_method_lock, num_req_method_merge, num_req_method_mkactivity, num_req_method_mkcalendar, num_req_method_mkcol,
+    num_req_method_mkredirectref, num_req_method_mkworkspace, num_req_method_move, num_req_method_options, num_req_method_orderpatch, 
+    num_req_method_patch, num_req_method_post, num_req_method_pri, num_req_method_propfind, num_req_method_proppatch, num_req_method_put,
+    num_req_method_rebind, num_req_method_report, num_req_method_search, num_req_method_trace, num_req_method_unbind, num_req_method_uncheckout,
+    num_req_method_unlink, num_req_method_unlock, num_req_method_update, num_req_method_updateredirectref;
+
+    /* Request protocol */
+    RRDSET *st_req_proto;
+    RRDDIM *dim_req_proto_http_1, *dim_req_proto_http_1_1, *dim_req_proto_http_2, *dim_req_proto_other;
+    collected_number num_req_proto_http_1, num_req_proto_http_1_1, num_req_proto_http_2, num_req_proto_other;
+
+    /* Request bandwidth */
+    RRDSET *st_bandwidth;
+    RRDDIM *dim_bandwidth_req_size, *dim_bandwidth_resp_size;
+    collected_number num_bandwidth_req_size, num_bandwidth_resp_size;
+
+    /* Request processing time */
+    RRDSET *st_req_proc_time;
+    RRDDIM *dim_req_proc_time_min, *dim_req_proc_time_max, *dim_req_proc_time_avg;
+    collected_number num_req_proc_time_min, num_req_proc_time_max, num_req_proc_time_avg;
+
+    /* Response code family */
+    RRDSET *st_resp_code_family;
+    RRDDIM *dim_resp_code_family_1xx, *dim_resp_code_family_2xx, *dim_resp_code_family_3xx, *dim_resp_code_family_4xx, *dim_resp_code_family_5xx, *dim_resp_code_family_other;
+    collected_number num_resp_code_family_1xx, num_resp_code_family_2xx, num_resp_code_family_3xx, num_resp_code_family_4xx, num_resp_code_family_5xx, num_resp_code_family_other;
+
+    /* Response code */
+    RRDSET *st_resp_code;
+    RRDDIM *dim_resp_code[501];
```

Comment:
please avoid magic numbers like `501`, for people that already studied `HTTP` protocol it is easy to understand your choice, but it can be complicated for people that did not work. It is better to use `define` and create a readable name.

Reply:
Done.